contracts/comptroller/Comptroller.sol

```diff
@@ -96,12 +96,22 @@ contract Comptroller is ComptrollerStorage, ComptrollerInterface {
 
   event VolumeDripDeactivated(
     uint256 indexed index,
-    address indexed prizeStrategy,
+    address indexed operator,
     address indexed measure,
     bool isReferral,
     uint256 activeIndex
   );
 
+  struct UpdatePair {
+    address operator;
+    address measure;
+  }
+
+  struct DripTokenBalance {
+    address dripToken;
+    uint256 balance;
+  }
+
   function initialize(address owner) public initializer {
```

Comment:
the "owner" variable name here shadows the "owner()" function of Ownable.  Recommend "_owner" here.